walkthroughs/1C-managing-and-protecting-using-api-mgmt/walkthrough.adoc

```diff
@@ -0,0 +1,381 @@
+        "openid"
+      ]
+    }
+  ],
+----
+.. Click *Create Service* at the bottom of the screen.
++
+. Show the Swagger UI in the Developer Portal.
+
+NOTE: The Developer Portal comes with a built-in ActiveDocs feature based on the Swagger UI library for API specification visualization. However, the built-in version (`v2.2.10`) doesn't work well with the OAuth flows, so we will use a more recent version of the library.
+
+.. Select *Audience* from the top navigation bar dropdown.
+.. Select *Developer Portal > Content* from the side navigation.
+.. Select the *Documentation* page in the built-in CMS.
+.. Replace the default content in the *Draft* tab with the following:
++
+[subs="attributes+"]
+----
+<style>
+#main-content .full > .container {background-color: white;}
+</style>
+
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/swagger-ui/3.23.11/swagger-ui.css" />
+<script src="https://cdnjs.cloudflare.com/ajax/libs/swagger-ui/3.23.11/swagger-ui-bundle.js"></script>
+
+<h1>Documentation</h1>
+<p>Use our live documentation to learn about the Orders API</p>
+<div id="swagger-ui"></div>
+<script type="text/javascript">
+  (function () {
+   
+    SwaggerUIBundle({
+      url: "{{ provider.api_specs.orders_api.url }}",
+      dom_id: '#swagger-ui',
+      presets: [
+        SwaggerUIBundle.presets.apis
+      ],
+      plugins: [
+        SwaggerUIBundle.plugins.DownloadUrl
+      ]
+    })
+  }());
+</script>
+----
+.. Click *Save* and *Publish*.
+.. Click *New Page* in the top right corner of the CMS.
+.. Enter `Swagger UI OAuth redirect` in the *Title* field.
+.. Enter `/oauth2-redirect.html` in the *Path* field.
+.. Select empty entry in the *Layout* field.
+.. In the *Draft* tab, copy and paste the contents of the link:https://raw.githubusercontent.com/swagger-api/swagger-ui/9253c0/dist/oauth2-redirect.html[`oauth2-redirect.html` page from Swagger UI, window="_blank"].
+.. Click *Create Page*.
+.. After the page is reloaded, click *Publish*.
+
+[type=verification]
+Click *Visit Portal* under the *Developer Portal* section on the side navigation and click *Documentation* in the top navigation. Can you see the visual representation of the Orders API specification?
+
+[type=verificationFail]
+Verify that you followed each step in the procedure above. If you are still having issues, contact your administrator.
+
+[time=10]
+== Make API calls through the Developer Portal
+
+. Log in to the Developer Portal:
+.. Go to the link:https://{user-sanitized-username}.{openshift-app-host}[3scale Developer Portal, window="_blank"].
+.. Close the side bar on the right, if it is open.
+.. Click *Sign in* in the top navigation bar.
+.. Enter `john` as *Username or Email* and `123456` as *Password*. These are the credentials of the _Developer_ account created by default for each tenant.
+.. Click *Sign in*.
++
+. Get the application credentials:
+.. Click *API Credentials* in the top navigation bar.
+.. Select *< Applications*.
```

Comment:
Is the `<` a typo?

Reply:
Well, actually it's not... 
<img width="326" alt="Applications" src="https://user-images.githubusercontent.com/1270328/66782539-31f40b00-eed6-11e9-99fa-88de43e69a6f.png">

But yeah, it's not really needed 😅